src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/Schema.java

```diff
@@ -104,7 +104,8 @@ public void remove() {
         DATE, // Date types
         BOOLEAN, // Boolean types
         BINARY, // Binary types
-        INTEGER_RANGE, FLOAT_RANGE, LONG_RANGE, DOUBLE_RANGE, DATE_RANGE; // Range types
+        INTEGER_RANGE, FLOAT_RANGE, LONG_RANGE, DOUBLE_RANGE, DATE_RANGE, // Range types
+        IP;
```

Comment:
Do you know if any correspondent type in JDBC specification? Because all types here will be mapped to JDBC types in driver.

Reply:
It's gonna be varchar

Reply:
Cool, if so I think we need to check if there is already mapping from IP to Varchar in JDBC driver or not.

Reply:
But the jdbc did't have the ip type mapped to varchar (or any other jdbc type). Need to add this mapping in JDBC. @dai-chen